# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/ipc/ContentParent.h
# Commit: 9ce1a2365aff
# Full Hash: 9ce1a2365afff092f9107a5f1ae1eca6a34dfce3
# Author: Nika Layzell <nika@thelayzells.com>
# Date: 2019-03-15 03:42:43
# Regressor Bug: 1532661
# File Overlap Count: 2
# Description:
#   Bug 1532661 - Part 4: Support initializing complete BrowsingContextGroups in a single op with an IPCInitializer struct, r=farre
#   
#   Depends on D22192
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D22193
# ==============================================================================

diff -r 58f77fa35eea -r 9ce1a2365aff dom/ipc/ContentParent.h
--- a/dom/ipc/ContentParent.h	Thu Mar 14 18:51:07 2019 +0000
+++ b/dom/ipc/ContentParent.h	Thu Mar 14 18:51:09 2019 +0000
@@ -610,8 +610,7 @@
   static bool IsInputEventQueueSupported();
 
   mozilla::ipc::IPCResult RecvAttachBrowsingContext(
-      BrowsingContext* aParentContext, BrowsingContext* aOpener,
-      BrowsingContextId aContextId, const nsString& aName);
+      BrowsingContext::IPCInitializer&& aInit);
 
   mozilla::ipc::IPCResult RecvDetachBrowsingContext(BrowsingContext* aContext,
                                                     bool aMoveToBFCache);